src/olympia/reviewers/tests/test_views.py

```diff
@@ -4873,39 +4894,6 @@ def setUp(self):
         self.addon.delete()
 
 
-class TestAbuseReports(TestCase):
```

Comment:
Everything in this class is already covered in `TestReview.test_abuse_reports` (and variants with that method name prefix)